tests/Types.php

```diff
@@ -8,6 +8,14 @@
 
 $pool = new MockPool();
 
+assertType('null', $pool->run(static function (): void {
```

Comment:
This assertion is misplaced.

Same as https://github.com/reactphp-parallel/infinite-pool/pull/60#discussion_r1910057904

You cannot take the value of a `void` expression, as it is has no possible value.

You can turn it into an expression by having a closure around it:

```diff
-assertType('null', $pool->run(static function (): void {
+assertType('Closure(): void', (fn () => $pool->run(static function (): void {
```